pkg/cluster/cluster_config.go

```diff
@@ -74,7 +74,10 @@ type Platform string
 // otherwise return "",err.
 func isSelfManaged(cli client.Client) (Platform, error) {
 	clusterCsvs := &ofapi.ClusterServiceVersionList{}
-	err := cli.List(context.TODO(), clusterCsvs)
+	// limit request to own namespace or since the operator is seen in all it can be too big
+	// ignore error, "" in this case is acceptable for functionality
+	ns, _ := GetOperatorNamespace()
```

Comment:
Should we use OperatorConditions or some other resources to identify the platform?

Reply:
Whatever works :)
Do you mean get the list of OperatorConditions and check the names for opendatahub-operator or rhods-operator similar way?

Reply:
OperatorExists() which calls OperatorConditions will only tell if it is ODH of RHOAI, does not say much if self-manager or managered.

Reply:
> OperatorExists() which calls OperatorConditions will only tell if it is ODH or RHOAI, does not say much if self-manager or managered.

But it will work here in this branch, won't it?

Reply:
> OperatorExists() which calls OperatorConditions will only tell if it is ODH or RHOAI, does not say much if self-manager or managered.

Thanks for pointing to the function, missed it